CRM/Upgrade/Incremental/php/SixTwo.php

```diff
@@ -29,6 +29,8 @@ class CRM_Upgrade_Incremental_php_SixTwo extends CRM_Upgrade_Incremental_Base {
    */
   public function upgrade_6_2_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Add column "civicrm_managed.checksum"', 'addColumn', 'civicrm_managed',
+      'checksum', "varchar(45) NULL COMMENT 'Configuration of the managed-entity when last stored'");
```

Comment:
Fun fact: we have a new helper for this: `alterSchemaField`. (you can see it in action 2 lines below this one)